README.md

```diff
@@ -140,6 +140,7 @@ You can build and/or use the maven coordinates to work with `azure-cosmosdb-spar
 
 | Spark | Scala | Latest version |
 |---|---|---|
+| 2.4.0 | 2.11 | [azure-cosmosdb-spark_2.4.0_2.11_3.0.3](https://search.maven.org/remotecontent?filepath=com/microsoft/azure/azure-cosmosdb-spark_2.4.0_2.11/3.0.3/azure-cosmosdb-spark_2.4.0_2.11-3.0.3-uber.jar)
```

Comment:
Please use this link instead - we will make sure that it points tot he "last known good" going forward.

https://aka.ms/CosmosDB_OLTP_Spark_2.4_LKG

Reply:
Thanks Fabian, so should I close the PR or or just remove all the SDK links and just point to the above link that you shared?

Reply:
I would just leave the links for Spark 2.1 / 2.2 and 2.3 untouched and replace the link for Spark 2.4 with the one above. Thanks!